pkg/terra/utils.go

```diff
@@ -68,3 +72,32 @@ func RawMessageStringIntToInt(msg json.RawMessage) (int, error) {
 	}
 	return strconv.Atoi(temp)
 }
+
+func MustAccAddress(addr string) cosmosSDK.AccAddress {
+	accAddr, err := cosmosSDK.AccAddressFromBech32(addr)
+	if err != nil {
+		panic(err)
+	}
+	return accAddr
+}
+
+// ConcatBytes appends a bunch of byte arrays into a single byte array
+func ConcatBytes(bufs ...[]byte) []byte {
+	return bytes.Join(bufs, []byte{})
+}
```

Comment:
This does not need to be a separate function! It doesn't do much and, afaict, it's only used once, in the function below.

Reply:
think im with @topliceanu, stick to the stdlib where we can

Reply:
sounds good :+1: